improver/nowcasting/optical_flow.py

```diff
@@ -712,13 +782,19 @@ def process(self, cube1, cube2, boxsize=30):
 
         Args:
             cube1 (iris.cube.Cube):
-                2D cube from (earlier) time 1
+                2D cube from (earlier) time 1 OR forecast cube from previous cycle
             cube2 (iris.cube.Cube):
-                2D cube from (later) time 2
+                2D cube from (later) time 2 OR observation matching forecast cube
```

Comment:
We have a from-cube and a to-cube in both cases that we are calculating the perturbation between. Perhaps these can be more neatly stated in these terms.

Reply:
I'm not sure exactly what you're suggesting.  Can you be more specific?
I can re-do the docstring but it might be tricky to rename the variables, they come up A LOT in this file.  However I can maybe do that if we come up with good ones.

Reply:
Sorry, I forgot to come back and expand this comment.
I was thinking of changing the doc-string, not the variable name.
Something like
`2D cube that advection velocity will be FROM`
`2D cube that advection velocity will be TO`

Reply:
OK.  I have added this in, but I've also kept the clarification that the second cube will always be the most recent observation, whereas `cube1` may be a forecast or a previous observation.